client/composants/popover/Popover.js

```diff
@@ -0,0 +1,154 @@
+import * as React from "react";
+import Popover from "@mui/material/Popover";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import { FlexContainer, Card, ContainerInformation } from "../../styles/global";
+
+export default function PopoverComposant() {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleClick = (e) => {
+    setAnchorEl(e.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? "simple-popover" : undefined;
+
+  return (
+    <div>
+      <Button aria-describedby={id} variant="contained" onClick={handleClick}>
+        Les Avis
+      </Button>
+      <Popover
+        id={id}
```

Comment:
follow last comment : 
```suggestion
        id={open ? "simple-popover" : undefined}
```